<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>200</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculate</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>31</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>x^2+</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdita">
    <property name="geometry">
     <rect>
      <x>38</x>
      <y>50</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditb">
    <property name="geometry">
     <rect>
      <x>104</x>
      <y>49</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>134</x>
      <y>50</y>
      <width>21</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>x+</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditc">
    <property name="geometry">
     <rect>
      <x>152</x>
      <y>49</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>192</x>
      <y>50</y>
      <width>21</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>=0</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelx1">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>21</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>x1=</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelx1res">
    <property name="geometry">
     <rect>
      <x>73</x>
      <y>90</y>
      <width>231</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelx2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>21</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>x2=</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelx2res">
    <property name="geometry">
     <rect>
      <x>74</x>
      <y>110</y>
      <width>231</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelmessage">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>191</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
